fxa/core.py

```diff
@@ -278,12 +285,7 @@ def get_email_status(self):
         return resp
 
     def verify_email_code(self, code):
-        body = {
-            "uid": self.uid,
-            "code": code,
-        }
-        url = "/recovery_email/verify_code"
-        self.apiclient.post(url, body)  # note: not authenticated
+        return self.client.verify_email_code(self.uid, code) # note: not authenticated
```

Comment:
> I guess it would be best to change the commit and not add a commit that just adds a space, right?

We can squash your commit if you want before merging them into master but I actually like to show that software is not perfect at first and that it improve with time. Showing this time is valuable to me because it is what makes history.